src/components/header/Header.component.tsx

```diff
@@ -7,39 +9,79 @@ import sun from '@/assets/icons/sun.svg';
 import userAddIcon from '@/assets/icons/user_Add.svg';
 import Button from '@/UI/button/Button.components.tsx';
 
+import styles1 from '../productCard/productCard.module.css';
 import styles from './header.module.css';
 
-const Header = () => (
-    <header className={styles.header}>
-        <img className={styles.mr_49} src={logo} alt="logo" />
-        <div className={styles.themes}>
-            <img src={sun} alt="sun" />
-            <div className={styles.verticalLine}></div>
-            <img src={moon} alt="moon" />
-        </div>
-        <nav className={styles.nav}>
-            <ul className={styles.nav__items}>
-                <li className={`${styles.nav__item} ${styles.mr_49}`}>
-                    <a className={`${styles.nav__link} ${styles.nav__link__active}`} href={' '}>
-                        About
-                    </a>
-                </li>
-                <li className={styles.nav__item}>
-                    <a className={styles.nav__link} href={' '}>
-                        Products
-                    </a>
-                </li>
-            </ul>
-        </nav>
-        <div className={styles.buttons}>
-            <img className={styles.cart} src={cart} alt="cart" />
-            <Button className={styles.mr_10} name="Login" backgroundColor="#333" icon={logOut} />
-            <Button name="Sign up" backgroundColor="#EF4934" icon={userAddIcon} />
-        </div>
-        <div className={styles.cart_and_burger}>
-            <img className={styles.cart} src={cart} alt="cart" />
-            <img className={styles.burger} src={burger} alt="burger" />
-        </div>
-    </header>
-);
+const Header = ({ amountCart, nav }: { amountCart: number; nav: (products: string) => void }) => {
+    const [activeNavItem, setActiveNavItem] = useState<string>('About');
+
+    const handleNavClick = (navItem: string) => {
+        nav(navItem);
+        setActiveNavItem(navItem);
+    };
+
+    return (
+        <header className={styles.header}>
+            <img className={styles.mr_49} src={logo} alt="logo" />
+            <div className={styles.themes}>
+                <img src={sun} alt="sun" />
+                <div className={styles.verticalLine}></div>
+                <img src={moon} alt="moon" />
+            </div>
+            <nav className={styles.nav}>
+                <ul className={styles.nav__items}>
+                    <li className={`${styles.nav__item} ${styles.mr_49}`}>
+                        <input
+                            type="button"
+                            className={`${styles.nav__link} ${activeNavItem === 'About' && styles.nav__link__active}`}
+                            value="About"
```

Comment:
![Screenshot 2024-05-08 at 17 34 55](https://github.com/lesiaturuta/fe-react-2024/assets/115080771/5c9aaaae-bc15-46cd-9d3a-a81d463ea627)

I am not sure that yellow color is suitable. And need to add cursor pointer for links i think.

Reply:
fixed